extern/storage-sealing/utils.go

```diff
@@ -55,3 +63,30 @@ func (m *Sealing) GetSectorInfo(sid abi.SectorNumber) (SectorInfo, error) {
 	err := m.sectors.Get(uint64(sid)).Get(&out)
 	return out, err
 }
+
+func collateralSendAmount(ctx context.Context, api interface {
+	StateMinerAvailableBalance(context.Context, address.Address, TipSetToken) (big.Int, error)
+}, maddr address.Address, cfg sealiface.Config, collateral abi.TokenAmount) (abi.TokenAmount, error) {
+	if cfg.CollateralFromMinerBalance {
+		avail := types.TotalFilecoinInt
+
+		if !cfg.DisableCollateralFallback {
```

Comment:
Seems correct as-is, but I feel like readability is improved by saying `if !cfg.DisableCollateralFallback { return big.Zero() }` at the start of this method.

Reply:
Maybe, but that isn't correct as when we have `!cfg.DisableCollateralFallback`, collateral still can be non-zero

Reply:
(unless you meant that the other way)

Reply:
i did mean that the other way